### ISSUE TITLE
Log in not persistant 

### ISSUE THREAD

**Issue by charlessteiner** [pierre-alain-b/rainloop-nextcloud#110]
Nextcloud 16.0.5, Rainloop 6.0.4, Firefox, PHP 7.2

With remember me checked, when I close tab and reopen nextcloud I need to log into rainloop. Nextcloud I don't need to log into again.

**Comment by pierre-alain-b** [comment]
Hi, can you tell us what you did to activate the auto-login? Have you set up email / password in the Additional setting section of the user settings?

**Comment by charlessteiner** [comment]
Thanks for the reply,

In additional settings for administration, I do have the box checked for  "Automatically login with Nextcloud user credentials".   https://postimg.cc/F7vmFccs

In additional settings for personal, the page is blank.

**Comment by pierre-alain-b** [comment]
You do not see this in Additional settings of the user?
![image](https://user-images.githubusercontent.com/386857/65818655-ba568880-e213-11e9-9162-862e8b835b69.png)

Rainloop is activated for the user?

**Comment by charlessteiner** [comment]
Thanks Pierre. it was user error. Everything is copacetic. 

I checked the Automatically login with user credentials after I had entered credentials in the user settings.  I didn't assume that the nextcloud user creds had to be the same as the email.

I suppose I thought that rainloop needed authorization to access the credentials previously saved. And I was a little confused by the blank Additional settings of the user page when when Automatically login with user credentials checked. 

Possibly for UX, a little notification in the Additional Settings of the user so there is no blank page. 

![image](https://user-images.githubusercontent.com/2821240/65819397-b7db3b00-e1e1-11e9-85a5-396717cb7484.png)
